content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -586,6 +595,12 @@ Each feature gate is designed for enabling/disabling a specific feature:
   (e.g. the cloud-controller-manager) in an HA cluster without downtime.
 - `CPUManager`: Enable container level CPU affinity support, see
   [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
+- `CPUManagerPolicyAlphaOptions`: This allows fine-tuning of CPUManager policies, experimental, Alpha-quality options
+  This feature gate will guard *a group* of CPUManager options whose quality level is alpha.
+  This feature gate will never graduate to beta or stable.
+- `CPUManagerPolicyBetaOptions`: This allows fine-tuning of CPUManager policies, experimental, Beta-quality options
+  This feature gate will guard *a group* of CPUManager options whose quality level is beta.
+  This feature gate will never graduate to stable.
```

Comment:
```suggestion
- `CPUManagerPolicyAlphaOptions`: This allows fine-tuning of CPUManager policies: experimental, Alpha-quality options.
  This feature gate guards *a group* of CPUManager options whose quality level is alpha.
  This feature gate will never graduate to beta or stable.
- `CPUManagerPolicyBetaOptions`: This allows fine-tuning of CPUManager policies: experimental, Beta-quality options.
  This feature gate guards *a group* of CPUManager options whose quality level is beta.
  This feature gate will never graduate to stable.
```

Reply:
Hi @sftim looks like you are already engaged on this. Assigning this PR to you.
/assign @sftim

Reply:
Fixed.